packages/rum/src/domain/record/observers/inputObserver.spec.ts

```diff
@@ -138,7 +133,7 @@ describe('initInputObserver', () => {
     const host = document.createElement('div')
     host.attachShadow({ mode: 'open' })
     const event = createNewEvent('input', { target: host, composed: true })
-    event.composedPath = () => [input, host, sandbox, document.body]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** I understand that we don't have the `sandbox` reference anymore, but this was implemented to mimic the browser behavior. Could this work?‏
```ts
event.composedPath = () => [input, host, input.parentElement, document.body]
```

Reply:
Sure!